app/helpers/shipit/github_url_helper.rb

```diff
@@ -55,5 +55,11 @@ def link_to_github_deploy(deploy)
       text = deploy.commit_range.map(&:short_sha).join('...')
       link_to(text, url, class: 'number')
     end
+
+    def link_to_github_range(stack, since_commit, until_commit)
```

Comment:
This looks unused now, can we 🔥 ?

Reply:
Weird, I could have sworn I deleted it. Maybe I ran a reset and took it with it. Will remove.